packages/threedimensional/src/scenes/World/index.ts

```diff
@@ -0,0 +1,196 @@
+//Commonjs import of three
+import { normalizeValue } from '../../utils';
+import * as THREE from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+
+class World {
+  renderer!: THREE.WebGLRenderer;
+  resolution!: THREE.Vector2;
+  camera!: THREE.PerspectiveCamera;
+  directionalLight!: THREE.DirectionalLight;
+  modelLoader!: GLTFLoader;
+  scene: THREE.Scene = new THREE.Scene();
+  clock: THREE.Clock = new THREE.Clock();
+  backgroundColor = "black";
+  transparentBackgroundColor = false;
+  hasToStopRendering = false;
+  canvasContainerId: string;
+  canvasSceneId: string;
+  observerOfCanvasContainer!: ResizeObserver;
+  controls!: OrbitControls;
+  trackMouseMovementBinded!: (event: MouseEvent) => void;
+  trackMouseScrollBinded!: (event: WheelEvent) => void;
+  threeDimensionalObjectOrSceneURL: string;
+  canvasContainer = {
+    width: 0,
+    height: 0,
+  }
+  CENTER_ORIGIN_AXES = {
+    x: 30,
+    y: -2,
+    z: 30,
+  }
+  CAMERA_POSITION = {
+    x: this.CENTER_ORIGIN_AXES.x,
+    y: this.CENTER_ORIGIN_AXES.y + 7,
+    z: this.CENTER_ORIGIN_AXES.z + 15,
+  }
+  CAMERA_ROTATION = {
+    x: -0.7,
+    y: 0,
+    z: 0,
+  }
+  constructor(canvasSceneId: string, canvasContainerId: string, backgroundColor?: string, threeDimensionalObjectOrScene?: Blob, transparentBackgroundColor?: boolean) {
+    const canvasContainer = document.getElementById(canvasContainerId) as HTMLElement
+    const canvasScene = document.getElementById(canvasSceneId) as HTMLCanvasElement
+    if (canvasScene && canvasContainer) {
+      if (backgroundColor) {
+        this.backgroundColor = backgroundColor
+      }
+      if (transparentBackgroundColor) {
+        this.transparentBackgroundColor = transparentBackgroundColor
+      }
+      this.modelLoader = new GLTFLoader();
+      if (threeDimensionalObjectOrScene) {
+        this.threeDimensionalObjectOrSceneURL = threeDimensionalObjectOrScene[0]
+      }
+      this.canvasSceneId = canvasSceneId
+      this.canvasContainerId = canvasContainerId
+      this.resolution = new THREE.Vector2(canvasScene.clientWidth, canvasScene.clientHeight)
+      this.camera = new THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 1000)
+      this.renderer = new THREE.WebGLRenderer({ canvas: canvasScene, antialias: true, alpha: true })
+      this.trackMouseMovementBinded = this.trackMouseMovement.bind(this)
+      this.trackMouseScrollBinded = this.trackMouseScroll.bind(this)
+      // this.controls = new OrbitControls(this.camera, this.renderer.domElement)
```

Comment:
@Rogerpeke97 Should we remove these coments? Or maybe add a ref to its use case?

Reply:
+1, we should remove all commented lines